apps/data-pipeline/ap-exams-importer/src/data.ts

```diff
@@ -0,0 +1,737 @@
+// https://apstudents.collegeboard.org/courses
+// https://catalogue.uci.edu/informationforprospectivestudents/undergraduateadmissions/#advancedplacementandinternationalbaccalaureatecredittext
+
+import type { APCoursesGrantedTree } from "@packages/db/schema";
+
+export const geCategories = [
+  "GE-1A",
+  "GE-1B",
+  "GE-2",
+  "GE-3",
+  "GE-4",
+  "GE-5A",
+  "GE-5B",
+  "GE-6",
+  "GE-7",
+  "GE-8",
+] as const;
+
+type APExam = {
+  catalogueName?: string;
+  rewards: {
+    acceptableScores: (1 | 2 | 3 | 4 | 5)[];
+    // units granted, most relevant for residency requirement
```

Comment:
that's crazy i didn't know PeterPortal factored this in

Reply:
it's in the table on the website, they might use this data in the future, and it would be more work to patch this column in later